### PR TITLE
REP-26: transition to Candidate

### PR DESCRIPTION
**Pull request by brucexc**
https://github.com/RSS3-Network/Whitepaper/pull/7 and https://github.com/RSS3-Network/Whitepaper/pull/8 for the Whitepaper are ready.

### CODE REVIEW

**Review by HenryQW — CHANGES_REQUESTED** [review]
run `pnpm format`. I have added the check in #31, do a rebase

File: REPs/REP-26.md [hunk 1]
```diff
@@ -37,3 +38,8 @@ The current staking mechanism, which mints a Chip for each unit of \$RSS3 staked
 ## Rationale
 
 The core of this proposal is to reduce the number of Chips minted to one for each staking transaction, regardless of the amount staked, where each Chip will represent the total value staked.
+
+## Reference Implementations
+
+1. Whitepaper: [RSS3-Network/Whitepaper#7](https://github.com/RSS3-Network/Whitepaper/pull/7)
+2. RSS3-Network-Contracts code update: <https://github.com/NaturalSelectionLabs/RSS3-Network-Contracts/pull/26>
```
Comment (HenryQW) [inline REPs/REP-26.md]: `Contracts update` should be enough

**Review by iavl — DISMISSED** [review]

File: REPs/REP-26.md (hunk 1 above)
Comment (HenryQW) [reply REPs/REP-26.md]: still unresolved

**Review by iavl — DISMISSED** [review]

File: REPs/REP-26.md [hunk 2]
```diff
@@ -37,3 +38,8 @@ The current staking mechanism, which mints a Chip for each unit of \$RSS3 staked
 ## Rationale
 
 The core of this proposal is to reduce the number of Chips minted to one for each staking transaction, regardless of the amount staked, where each Chip will represent the total value staked.
+
+## Reference Implementations
+
+1. Whitepaper: [RSS3-Network/Whitepaper#7](https://github.com/RSS3-Network/Whitepaper/pull/7) and [RSS3-Network/Whitepaper#8](https://github.com/RSS3-Network/Whitepaper/pull/8)
```
Comment (HenryQW) [inline REPs/REP-26.md]: change to list:
1. Whitepaper
    1. [RSS3-Network/Whitepaper#7](https://github.com/RSS3-Network/Whitepaper/pull/7)
    1.  [RSS3-Network/Whitepaper#8](https://github.com/RSS3-Network/Whitepaper/pull/8)

**Review by HenryQW — CHANGES_REQUESTED** [review]

**Review by iavl — APPROVED** [review]

**Comment by brucexc** [comment]
Pending contract upgrades, attach updated contracts in the Reference Implementations section.

**Review by HenryQW — APPROVED** [review]

**Pull request merged**